Update docs per rebar3_hex v7 release (#1098)
hexpm/hexpm: lib/hexpm_web/templates/docs/rebar3_publish.html.md

  <dd>A map where the key is a link name and the value is the link URL. Optional but highly recommended.</dd>
  <dt><code>files</code></dt>
  <dd>A list of files and directories to include in the package. Defaults to standard project directories, so you usually don't need to set this property.</dd>

>   <dt><code>include_paths</code></dt>
>   <dd>A list of paths containing files you wish to include in a release.</dd>
>   <dt><code>exclude_paths</code></dt>
>   <dd>A list of paths containing files you wish to exclude in a release.</dd>

Reviewer: ```suggestion
  <dt><code>include_paths</code></dt>
  <dd>A list of paths containing files you wish to include in the package.</dd>
  <dt><code>exclude_paths</code></dt>
  <dd>A list of paths containing files you wish to exclude in the package.</dd>
```

  <dt><code>exclude_patterns</code></dt>
  <dd>A list of regular expressions used to exclude files that may have been accumulated via files and include_paths and standard project paths.</dd>
  <dt><code>build_tools</code></dt>
  <dd>List of build tools that can build the package. It's very rare that you need to set this.</dd>

> 

Reviewer: ```suggestion
```

```

#### Documentation 

rebar3_hex by default expects you to configure a documentation provider. We recommend using 
[rebar3_ex_doc](https://hexdocs.pm/rebar3_ex_doc/) for publishing documentation along with your package for a consistent 
format and style on [hex.pm](https://hex.pm/). 

> 

Reviewer: ```suggestion
```

```erlang 

> ex_doc, [

Reviewer: ```suggestion
{ex_doc, [
```

    {source_url, <<"https://github.com/namespace/your_app">>},
    {extras, [<<"README.md">>, <<"LICENSE">>]},
    {main, <<"readme">>}
]}.

{hex, [{doc, ex_doc}]}.
```

> Alternatively, or on Erlang versions older than OTP-24, you can use the edoc provider that ships with rebar3 : 

Reviewer: ```suggestion
Alternatively, or on Erlang versions older than OTP-24, you can use the edoc provider that ships with rebar3: 
```
